system-addon/content-src/components/Base/Base.jsx

```diff
@@ -94,7 +94,7 @@ export class BaseContent extends React.PureComponent {
 
     const shouldBeFixedToTop = PrerenderData.arePrefsValid(name => prefs[name]);
 
-    const outerClassName = `outer-wrapper ${Theme.className}${shouldBeFixedToTop ? " fixed-to-top" : ""} ${prefs.enableWideLayout ? "wide-layout-enabled" : "wide-layout-disabled"}`;
+    const outerClassName = `outer-wrapper${Theme.className ? ` ${Theme.className}` : ""}${shouldBeFixedToTop ? " fixed-to-top" : ""} ${prefs.enableWideLayout ? "wide-layout-enabled" : "wide-layout-disabled"}`;
```

Comment:
eh. mrhm.m.m.... I know this is how we do it elsewhere.... What if we just sneak in a simple filter/join..
```js
const outerClassName = [
  "outer-wrapper",
  Theme.className,
  shouldBeFixedToTop && "fixed-to-top",
  prefs.enableWideLayout ? "wide-layout-enabled" : "wide-layout-disabled"
].filter(v => v).join(" ");
```

I thought @k88hudson had written something like this somewhere already…

Maybe we sneak it in here and add it to some `content-src/lib` later ;)